pytest.ini

```diff
@@ -1,4 +1,8 @@
 #Configuration file for py.test
 [pytest]
 python_files=Test_*.py assert*.py
-addopts = -rx -v --color=yes --tb=long --ignore=tests --cov=. --cov-report term-missing
+# By default, we do not run on Core/Security/test
+# The reason here is that we do nasty things with the pythonpath
+# in order to make sure that M2Crypto and pyGSI do not step
+# on each other's feet
+addopts = -rx -v --color=yes --tb=long --ignore=tests --ignore=Core/Security/test --cov=. --cov-report term-missing
```

Comment:
Can we somehow run these tests anyway? At least part of it, maybe in a different step of the pipeline?

Reply:
they are ran separately bellow